pkg/virt-handler/vm.go

```diff
@@ -1017,7 +1016,7 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 
 	cache.WaitForCacheSync(stopCh, c.domainInformer.HasSynced, c.vmiSourceInformer.HasSynced, c.vmiTargetInformer.HasSynced, c.gracefulShutdownInformer.HasSynced)
 
-	go c.heartBeat(c.heartBeatInterval, stopCh)
+	go c.heartBeat.Run(c.heartBeatInterval, stopCh)
```

Comment:
what happens to the "done" channel this function returns?

Reply:
I use it right now for unit tests to wait for go routines. We could make use of it for more structured shutdowns in the future, but I need it right now for testing only. In production the channel gets closed and nothing else happens.